quality_control_stock/models/stock_picking.py

```diff
@@ -36,6 +24,23 @@ def _compute_count_inspections(self):
     failed_inspections = fields.Integer(
         compute="_compute_count_inspections", string="Inspections failed")
 
+    @api.depends('qc_inspections_ids', 'qc_inspections_ids.state')
+    def _compute_count_inspections(self):
+        data = self.env['qc.inspection'].read_group([
+            ('id', 'in', self.mapped('qc_inspections_ids').ids),
+        ], ['picking_id', 'state'], ['picking_id', 'state'], lazy=False)
+        picking_data = defaultdict(dict)
```

Comment:
Why do you need `defaultdict`? I think regular dict works as well with `setdefault`